dist/ui-toolkit.standalone.js

```diff
@@ -28436,10 +28435,104 @@ module.exports = require('./code/index');
 
 
 },{"./code/index":255}],259:[function(require,module,exports){
+module.exports = require('./views/stepperView.jsx');
+
+
+},{"./views/stepperView.jsx":261}],260:[function(require,module,exports){
+'use strict';
+
+var React = require('react');
+var Button = require('../../../button');
+var Input = require('../../../input');
+
+module.exports = function() {
+
+  return (
+    React.createElement("div", {className: "component-stepper"}, 
+      React.createElement("span", {className: "button-container"}, 
+        React.createElement(Button, {handleClick: this.decrement, disabled: !this.canDecrement()}, this.props.decrementDisplayString)
+      ), 
+
+      React.createElement(Input, {type: "text", key: this.state.value, readOnly: true}, this.state.value.toString()), 
+
+      React.createElement("span", {className: "button-container"}, 
+        React.createElement(Button, {handleClick: this.increment, disabled: !this.canIncrement()}, this.props.incrementDisplayString)
+      )
+    )
+  );
+};
+
+
+},{"../../../button":210,"../../../input":237,"react":197}],261:[function(require,module,exports){
+'use strict';
+
+var React = require('react');
+
+module.exports = React.createClass({displayName: "exports",
+
+  propTypes: {
+    value: React.PropTypes.number,
+    onChange: React.PropTypes.func,
+    minValue: React.PropTypes.number,
+    maxValue: React.PropTypes.number,
+    incrementDisplayString: React.PropTypes.string,
+    decrementDisplayString: React.PropTypes.string
+  },
+
+  getDefaultProps: function() {
+    return {
+      value: 0,
+      incrementDisplayString: '+',
+      decrementDisplayString: '-',
+      onChange: function() {}
+    };
+  },
+
+  getInitialState: function() {
+    return {
+      value: this.props.value
```

Comment:
consider that checked! Get rid of state completely here, it simplifies your code. Then one more general point, if you run coverage you'll see if your new test covers all the possibilities - right now it does not.
```
npm run coveralls;
open coverage/lcov-report/index.html 
```
some of the other tests do not cover enough either, that's not your problem and I will fix that, but you should aim for 100% coverage with your stepper test. When you get rid of state you will find you have less code to cover so your test might even get simpler.
`coveralls` is the name of the npm script here though it's likely to change to `npm run coverage` soon if we standardise...